kong/runloop/plugin_servers/pb_rpc.lua

```diff
@@ -386,13 +387,19 @@ end
 
 
 function Rpc:handle_event(plugin_name, conf, phase)
-  local instance_id = self.get_instance_id(plugin_name, conf)
+  local instance_id, err = self.get_instance_id(plugin_name, conf)
```

Comment:
We can define `local res, instance_id, err`, then use them to get a relatively clean code.